theme/templates/accounts/sign_up_form.html

```diff
@@ -219,7 +219,7 @@
 
     $('.tagsinput').tagsInput({
       interactive: true,
-      placeholder: "Organization(s)",
+      placeholder: "Organization(s) *",
```

Comment:
What prompted this change?

Reply:
Dave was testing the fix I made for profile pic upload. He was trying to create a new user account and noticed that some of the required fields are not marked visually marked as required. He asked me to fix that as part of this PR.

Reply:
I see now, thanks.